### ISSUE TITLE
omr-admin status failed VPS

### ISSUE THREAD

**Issue by runfortheapples** [Ysurac/openmptcprouter#768]
## Expected Behavior

 omr-admin service on vps status to be active.

## Actual Behavior
omr-admin service on vps current status is failed:
![image](https://user-images.githubusercontent.com/29462618/71545388-c117a780-298a-11ea-83b3-36544f936eaa.png)

OenMPTCProuter can't contact Server Admin Script:
![image](https://user-images.githubusercontent.com/29462618/71545401-e9070b00-298a-11ea-8080-f825615aefbb.png)

## Steps to Reproduce the Problem

  1. upgrade openmptcprouter from 0.53.0 to 0.53.1
  2. use vps script to upgrade server from 0.1004 to 0.1005

## Specifications

  - OpenMPTCProuter version: 0.53.1
  - OpenMPTCProuter VPS version: 0.1005 Ubuntu 18.04
  - OpenMPTCProuter platform: RPI4(64-bit)

Screenshot of syslog:
![image](https://user-images.githubusercontent.com/29462618/71545487-d4774280-298b-11ea-9f90-30dbdd277feb.png)

**Comment by Ysurac** [comment]
```pip3 install pyjwt``` should solve the issue, but this should have be installed via the script.

**Comment by runfortheapples** [comment]
![image](https://user-images.githubusercontent.com/29462618/71545576-2bc9e280-298d-11ea-89b3-5c0e35c74588.png)

Seems to be already installed.

**Comment by Ysurac** [comment]
Strange... I will check on a Ubuntu 18.04 if I can reproduce the issue...

**Comment by Ysurac** [comment]
I'm not able to reproduce the issue on Ubuntu 18.04 from a fresh install with latest script.

**Comment by runfortheapples** [comment]
Ran a fresh instance of ubuntu18.04 and had the same error. Got the service to start by doing `pip3 install jwt` but still had the error that the openmptcprouter couldn't contact the VPS. 
In the end I started a fresh instance of debian9 and ran the script. 
Router can now contact the VPS:
![image](https://user-images.githubusercontent.com/29462618/71548862-ad813680-29b4-11ea-9da4-6a871ee0a491.png)

Thank you for your time @Ysurac
